### DISCUSSION TITLE
Line length marker / Font size for anything not terminal/buffer

### DISCUSSION THREAD

**Discussion by r-sergeant** [zed-industries/zed#6578] [Feature Requests]
Love Zed so far (2 weeks in). It's fast and easily configurable... Just 2 things I can't really find how to do (I don't know if it is possible, but it would be nice for me to have):

* I usually have fairly small font sizes when working on my laptop away from my desk. For buffer/terminal I have it set to 11. It would be nice though if I could set the font size for other things smaller too (project, tabpage titles, navigation indicators). Is this possible?
* I have some legacy formats to deal with and it would be nice if I could set line markers at certain positions (e.g. column 15, 80, 90, 100). Setting only one would already be helpful. Again I can't find how to do this (if it is possible)?

I've searched it here and in the manual but without succes.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
